### ISSUE TITLE
"Add new feed or article" dialogue: Add filter

### ISSUE THREAD

**Issue by touwys** [yanus171/Handy-News-Reader#785]
If there is no compelling reason not to, please add the **"Filter"** operation to the dialogue shown:

![Screenshot_20220425_083331](https://user-images.githubusercontent.com/65730626/165033752-848a5ccb-ca05-4482-898e-c686d3f0a427.png)

**Comment by yanus171** [comment]
Filters are accessible on edit feed action, after the feed was created. Is 
it not enough?

**Comment by touwys** [comment]
No, because for some reason, the newly added feed disappears from view immediately after being added. Instead, user has to find it in the Feed Group to which it was added, and then enter the setup window to add the filter. (This behaviour is always inconsistent and unexpected, which means it is impossible to provide a screen video of the issue.)
